### ISSUE TITLE
set marginRight twice, the view hang there 

### ISSUE THREAD

**Issue by changfuguo** [facebook/react-native#3806]
here i set style marginRight twice

       coverWrapper:{
          marginRight:5,
          width:60,
          height:93,
          flexDirection:'column',
          marginRight:20
      },

then the Interface is white and  hang there，when i remove  '  marginRight:5,' , then ok
i guess we can not set a style twice, or it is bug ???

**Comment by mangogogos** [comment]
It's just not a defined behaviour. I ran into the same issue ( #3584 ) and was recommended to use eslint which has improved efficiency by making it trivial to find such typos and mistakes! Just make sure to grab the babel parser and never make this mistake again ( no-dupe-keys: 1 ) which should be set already in the .eslintrc file in the root of this repo. Out of curiosity how are you debugging your application? When I encountered this running in the chrome debugger actually fixed this issue for me and allowed me to keep working so it wasn't until much later that I found my mistake
